### DISCUSSION TITLE
Couldn't find my Student Subrisciption but It's already valid and the coupon is steal valid until march 2023

### DISCUSSION THREAD

**Discussion by basileroutier** [community/community#41375] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello,
I have a problem since today, I can't connect to copilot without it showing me the following error:
"No access to GitHub Copilot found."
The problem is that I have checked my benefits and my student pack is still active until March 2023.

I tried to reactivate my coupon but as it is still valid, I do not see where the problem can come from especially since I had never encountered this problem before...

**Comment by codingforalife** [comment]


**Reply by christosalexopoulos** [reply]
same here

**Comment by onecrazygenius** [comment]
I had this issue a few minutes ago as well

**Reply by onecrazygenius** [reply]
On vscode says invalid copilot token: missing token: 403. GitHub copilot could not connect to server. Extension failed: "Thank you for participating...." on vscode output

**Comment by EdenBendheim** [comment]
Same issue as well

**Comment by xlucasbitencourt** [comment]
I have the same issue. I have a valid student subscription, and can download others apps, but copilot isn't working.

**Comment by GamingLiamStudios** [comment]
Same here. Was just about to file a support ticket lol.

**Comment by Sultron** [comment]
I have the same issue as well.

**Comment by 18jad** [comment]
The same issue with my vscode copilot, I have a valid student pack, and I tried to sign out and log back in but the problem persists. It seems like copilot has some issues currently, same thing happened 2 weeks ago when they pushed an update.

**Reply by onecrazygenius** [reply]
This is only acceptable if I can say "Hey GitHub" after this update lmao

**Reply by 18jad** [reply]
Man life without copilot feels so lonely lmao gotta stop using it

**Reply by onecrazygenius** [reply]
Yeah, it's surprising how pain free it makes doing those boring steps. Please come back with the milk copilot I miss you

**Reply by AkuVolts** [reply]
can't believe how used to it I am now coding feels slow and wrong without it

**Comment by azaryabernard** [comment]
Hey I have the same issue as well, when opening the copilot page in Setting, it doesn't have the option to enable Github Copilot

**Comment by rafael-rm** [comment]
I'm having the same problem.

**Comment by Gasppo** [comment]
Yeah just lost access today as well and get the normal billing section if I try to enable it
![image](https://user-images.githubusercontent.com/22486381/206879958-e40e594d-be41-47a9-b2ae-3745ce2df37a.png)

**Comment by Kseen715** [comment]
Same issue...

**Comment by MrOplus** [comment]
it seems the bad deployment from 15-16 days ago still affects users  
check this https://github.com/orgs/community/discussions/40037

**Reply by 18jad** [reply]
It's been 16 days since no one was affected, looks like they made a new deploy now

**Reply by MrOplus** [reply]
yes, maybe the new deployment broke up some things like our copilot subscription, GitHub eventually sends emails to some users about the subscription renewal! it seems they are working on a fix :/

**Reply by 18jad** [reply]
Idk if they are really working on it but [status page](https://www.githubstatus.com/) shows that everything is running properly

**Reply by MrOplus** [reply]
based on the last open discussions it seems they are validating the licenses manually! lol

**Reply by 18jad** [reply]
okay now it's fixed and working fine

**Comment by RagingGam0r** [comment]
Got same issue with GitHub co-pilot w/ Visual Studio
```
Please open https://github.com/login/device and enter the device activation code XXXX-XXXX that has been copied to the clipboard.
You don't have access to GitHub Copilot.

Please open https://github.com/login/device and enter the device activation code XXXX-XXXX that has been copied to the clipboard.
You don't have access to GitHub Copilot.

Please open https://github.com/login/device and enter the device activation code XXXX-XXXX that has been copied to the clipboard.
You don't have access to GitHub Copilot.
```

And trying to add again says to pay / use free trial:

![image](https://user-images.githubusercontent.com/42112694/206879999-cc3c35e5-da84-47ba-b595-b86ab0ed714a.png)

From looking seems to be a problem reported around ~16 days ago, and seems to have been fixed at that time but has began to reoccur.
https://github.com/orgs/community/discussions/40037

**Reply by RagingGam0r** [reply]
Seems to have been fixed already (at least for me)
Re verifying in Visual Studio now works. (Was down for about 30~mins for me)

**Reply by joepalexander** [reply]
Can confirm - Fixed for me too!

**Comment by AkuVolts** [comment]
I have the same issue this is so sad.

**Comment by Pop101** [comment]
Same issue for me. I've got the student dev pack active, no renewal necessary, but copilot hits me with the "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

**Comment by 18jad** [comment]
[Github status](https://www.githubstatus.com/) showing that everything is working fine...

**Comment by erenberkaydinc** [comment]
I have the same issue , I have student developer pack, and everything's verified and still valid and I can't use GitHub Copilot, it always redirects me to paying page for copilot

**Comment by VidunderGunder** [comment]
Same problem here as well 💔

**Comment by eliac7** [comment]
Guys, probably just got fixed. Check your profile [here](https://github.com/settings/billing)

![2022-12-11 02_01_55-Billing](https://user-images.githubusercontent.com/26083840/206880284-e68b6703-bd0d-4b17-8846-0a8e2824d47b.png)

**Reply by azaryabernard** [reply]
Yes, now it's working!

**Reply by 18jad** [reply]
yeyyyyy finally

**Reply by VidunderGunder** [reply]
You're right, it's fixed now.

Made me realise how Copilot-dependent I've become, though.

**Reply by AkuVolts** [reply]
yayyy i feel whole again!!

**Comment by Akash1134** [comment]
👋 Hi there, Thank you everyone for taking the time to report this issue.

Apologies for that temporary issue on our end. Please retry logging in to your IDE plugin, and Copilot should work as expected. 

If you run into any further issues, please reply back in detail with any useful screenshots.

**Comment by renko80** [comment]
Hi  @eliac7 , @Akash1134     I am facing the same problems, 

![image](https://user-images.githubusercontent.com/120383335/208380387-7e40fd1a-7cc8-420e-8e66-42c562f4eacf.png)


And I enter in the profile as @eliac7 suggested,   but  I dont find 

![image](https://user-images.githubusercontent.com/120383335/208380552-0d59e4a1-94d6-4a00-8912-465a1ddc6bff.png)


In my account 

I have the following 

![image](https://user-images.githubusercontent.com/120383335/208381146-379e150f-f709-41ef-b313-2470e503a3ef.png)


And when I click over "Enable Github Copilot", it appears to me  Payment Prices

![image](https://user-images.githubusercontent.com/120383335/208381454-de726e98-a5cb-4194-8348-500d1c84a404.png)



I only want to use it, as trial by the moment. Dont need any reference to future payments.  Is this possible because I mistake when creating the account,  because this possibility only happens when you select the student account?

Many thanks in advance for your support!
